### ISSUE TITLE
Can't Share A Group in Contacts

### ISSUE THREAD

**Issue by jjaiso** [nextcloud/contacts#2290]
### Describe the bug
......
......

**To Reproduce**
1. Go to Contacts App
2. Click on 'Create New Group +'
3. Try to share contact group
4. There is no way to share the group to someone within the same NextCloud instance

**Expected behavior**
Be able to create a 'global contact list' and share it amongst users or all users within the same instance

**Actual behavior**
I can't share a contact group list

**Screenshots**
![Screen Shot 2021-06-18 at 4 28 50 PM](https://user-images.githubusercontent.com/83552298/122623844-8a3bc280-d052-11eb-8849-cae406b9de81.png)

### Server configuration

**Operating system**: Linux

**Web server:** Apache

**Database:** MySQL 8.0.25

**PHP version:** 7.4

**Nextcloud version:** 20.0.10

**Contacts version:** 3.5.1

**Fresh Install**

**Signing status:**
```
No errors have been found.
```

**List of activated apps:**
```
Enabled:
 - accessibility: 1.6.0
 - activity: 2.13.4
 - aisonetauthapi: 1.0.1
 - analytics: 3.5.1
 - apporder: 0.12.0
 - bruteforcesettings: 2.2.0
 - calendar: 2.2.2
 - cloud_federation_api: 1.3.0
 - comments: 1.10.0
 - contacts: 3.5.1
 - contactsinteraction: 1.1.0
 - dashboard: 7.0.0
 - dav: 1.16.2
 - deck: 1.2.7
 - federatedfilesharing: 1.10.2
 - files: 1.15.0
 - files_antivirus: 3.2.1
 - files_pdfviewer: 2.0.1
 - files_retention: 1.9.0
 - files_rightclick: 0.17.0
 - files_sharing: 1.12.2
 - files_trashbin: 1.10.1
 - files_videoplayer: 1.9.0
 - firstrunwizard: 2.9.0
 - forms: 2.2.4
 - impersonate: 1.7.0
 - logreader: 2.5.0
 - lookup_server_connector: 1.8.0
 - mail: 1.9.5
 - notes: 4.0.4
 - notifications: 2.8.0
 - oauth2: 1.8.0
 - password_policy: 1.10.1
 - photos: 1.2.3
 - privacy: 1.4.0
 - provisioning_api: 1.10.0
 - quota_warning: 1.9.1
 - recommendations: 0.8.0
 - richdocuments: 3.8.2
 - serverinfo: 1.10.0
 - settings: 1.2.0
 - sharebymail: 1.10.0
 - spreed: 10.0.8
 - support: 1.3.0
 - survey_client: 1.8.0
 - systemtags: 1.10.0
 - tasks: 0.13.6
 - text: 3.1.0
 - theming: 1.11.0
 - twofactor_backupcodes: 1.9.0
 - twofactor_totp: 5.0.0
 - twofactor_u2f: 6.1.0
 - updatenotification: 1.10.0
 - user_status: 1.0.1
 - viewer: 1.4.0
 - weather_status: 1.0.0
 - workflowengine: 2.2.0
Disabled:
 - admin_audit
 - encryption
 - federation
 - files_external
 - files_versions
 - maps
 - news
 - nextcloud_announcements
 - user_ldap
```

**Nextcloud configuration:**
```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "office.controlmywebsite.com"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "skeletondirectory": "",
    "dbtype": "mysql",
    "version": "20.0.10.1",
    "overwrite.cli.url": "https:\/\/office.controlmywebsite.com\/",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "maintenance": false,
    "simpleSignUpLink.shown": false,
    "login_form_autocomplete": true,
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpmode": "smtp",
    "mail_sendmailmode": "smtp",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauthtype": "LOGIN",
    "mail_smtpauth": 1,
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "587",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpsecure": "tls",
    "filelocking.enabled": true,
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "memcache.distributed": "\\OC\\Memcache\\Redis",
    "memcache.local": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 6379,
        "timeout": 3,
        "dbindex": 0
    },
    "theme": "",
    "loglevel": 2
}

```

### Client configuration
Using Safari

**Operating system:**
macOS Catalina

**Comment by jjaiso** [comment]
Hello,

Quick update on this.  I found how to share a contact within a group.  Is there a chance to set the default address book for each Group?  This avoids having to select the address book at the individual contact level, and you can just add contacts to their respective Groups and not have to select the address book every time.

Thanks,

**Comment by skjnldsv** [comment]
Hey, this is not supported. Groups cannot be shared

**Comment by jjaiso** [comment]
Thank you for the response.

Is there a chance setting a default address book for a group will be featured?

Thanks,

— 
Julio Sandoval
Customer Success & Operations
Affordable Internet Services Online, Inc. 
https://www.aiso.net
Schedule A Call - https://aisonet.bookafy.com/

Toll Free:  1-800-781-9004 ext. 102
Local:       1-951-928-8190 ext . 102
Solar Powered Web Site Hosting, Dedicated Servers, Co-Location, Mailing Lists,
Content Delivery (CDN), Virtual Desktops, Web Site Design & More.

Send large files for free using solar power at https://www.solarsendit.net
Follow us on Twitter at https://twitter.com/#!/aisonet
Follow us on Facebook at https://www.facebook.com/AISOnetGreen

Disclaimer

The information contained in this communication from the sender is confidential. It is intended solely for use by the recipient and others authorized to receive it. If you are not the recipient, you are hereby notified that any disclosure, copying, distribution or taking action in relation of the contents of this information is strictly prohibited and may be unlawful.

**Comment by skjnldsv** [comment]
I'm not sure I understand your question :thinking:

**Comment by jjaiso** [comment]
> I'm not sure I understand your question 🤔

I created a Global Address Book that I shared with my whole group.

![Screen Shot 2021-06-25 at 9 52 15 AM](https://user-images.githubusercontent.com/83552298/123459396-17bf6b00-d59b-11eb-8c21-0f1a406fa033.png)

I created a contact group called Global Address List
![Screen Shot 2021-06-25 at 9 53 01 AM](https://user-images.githubusercontent.com/83552298/123459465-2ad23b00-d59b-11eb-88fd-cbe6e3f2efe1.png)

If I select the Global Address List group and add a new contact, it always defaults to the 'Contacts' Address Book.  Can we find a way to set a default 'Address Book' for a Contact Group?  This would avoid having to change the address book every single time.

![Screen Shot 2021-06-25 at 9 53 32 AM](https://user-images.githubusercontent.com/83552298/123459601-5ce39d00-d59b-11eb-948a-c58e9425d0f1.png)

Thank you so much for your time and attention!
